cyences_app_for_splunk/default/alert_actions.conf

```diff
@@ -32,4 +33,5 @@ param.email_to_default =
 param.cyences_severities = critical
 param.email_to_include = 
 param.email_to_exclude = 
+param.subject_prefix = [N/A] 
```

Comment:
Because its a generic configuration, we need to put into some other file I think.

We can use not only for email but other places, so lets not store here.